apps/services/regulations-admin-backend/seeders/20210707133000-seed_draft_regulation.js

```diff
@@ -66,7 +66,7 @@ const draftRegulationSeed = [
       '<h3 class="article__title">1. gr.</h3>\n<p>Ákvæði til bráðabirgða I í reglugerðinni orðist svo:</p>\n<p>\n\tÞrátt fyrir ákvæði 5. mgr. 6. gr. reglugerðarinnar er heimilt að endurnýja\n\tnotendaleyfi án þess að viðkomandi hafi aflað sér viðeigandi endurmenntunar,\n\ttil 31. desember 2021.\n</p>\n<h3 class="article__title">2. gr.</h3>\n<p>\n\tReglugerð þessi er sett með stoð í 15. tölul. 11. gr. efnalaga nr. 61/2013.\n</p>\n<p>Reglugerðin öðlast þegar gildi.</p>\n<p class="Dags" align="center">\n\t<em>Umhverfis- og auðlindaráðuneytinu, 9. febrúar 2018.</em>\n</p>\n<p align="center">\n\t<strong>Guðmundur Ingi Guðbrandsson.</strong>\n</p>\n<p align="right">\n\t<em>Sigríður Auður Arnardóttir.</em>\n</p>\n',
     drafting_notes: '',
     ideal_publish_date: '2021-08-08',
-    ministry_id: 'avnsr',
+    ministry: 'avnsr',
```

Comment:
These are nonsense example/mock values, BTW, but its harmless, since the `ministry` field is exactly supposed to support arbitrary (even seemingly nonsensical) values during the drafting/editing phase.